src/Insights/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If I remove this file I get a conflict for Newtonsoft.Json.dll.
Microsoft.WindowsAzure.Storage 4.0.1, and all versions I have tried, needs Newtonsoft.Json.dll version 4.5.0.0. All our other projects use version 6.0.0.0.